react/src/pages/RoutingListPage.tsx

```diff
@@ -265,7 +266,10 @@ const RoutingListPage: React.FC<RoutingListPageProps> = () => {
           <Button
             type="primary"
             icon={<SettingOutlined />}
-            disabled={(endpoint?.desired_session_count || 0) < 0}
+            disabled={
+              (endpoint?.desired_session_count || 0) < 0 ||
```

Comment:
User can modify `desired_session_count` later. Leave just filtering the status.
```suggestion
```

Reply:
@agatha197 I just removed desired_session_count.